Telecom/main_window.cs

```diff
@@ -64,6 +66,17 @@ protected override void RenderWindowContents(int window_id) {
           open_contracts_.Add(contract, false);
         }
       }
+      var connectionLabelStyle = Style.Multiline(GUI.skin.label);
+      var okStyle = new GUIStyle(Style.RightAligned(GUI.skin.label)){
+        focused = {
+            textColor = XKCDColors.Lime
+        },
+        normal = {
+            textColor = XKCDColors.Lime
+        }
+      };
```

Comment:
```suggestion
      var okStyle = Style.RightAligned(GUI.skin.label) ;
```


Let’s not create _ad hoc_ new styles with new colours in the middle of the rendering like that. Just use the error style for disconnected, and the default colour for OK.